extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -575,6 +668,15 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 
 	public async getBlobContainerValues(subscription: azureResource.AzureResourceSubscription, storageAccount: StorageAccount): Promise<azdata.CategoryValue[]> {
 		let blobContainerValues: azdata.CategoryValue[] = [];
+		if (!this._azureAccount || !subscription || !storageAccount) {
```

Comment:
This doesn't seem related to this change, was this intentional?

Reply:
This was just a runtime error I caught